src/pybind/rados/rados.pyx

```diff
@@ -3889,6 +3890,31 @@ returned %d, but should return zero on success." % (self.name, ret))
         with nogil:
             rados_write_op_omap_clear(_write_op.write_op)
 
+    @requires(('write_op', WriteOp), ('key_begin', str_type), ('key_end', str_type))
+    def remove_omap_range2(self,write_op, key_begin, key_end):
+        """
+        Remove key/value pairs from an object whose keys are in the range
+        [key_begin, key_end)
+
+        :para write_op: write operation object
+        :type write_op: WriteOp
+        :param key_begin: the lower bound of the key range to remove
+        :type key_begin: str
+        :param key_end: the upper bound of the key range to remove
+        :type key_end: str
+        """
+        key_begin = cstr(key_begin, 'key_begin')
+        key_end = cstr(key_end, 'key_end')
+        cdef:
+            WriteOp _write_op = write_op
+            char* _key_begin = key_begin
+            size_t key_begin_len = len(key_begin)
```

Comment:
since key_begin is a string here specifying a number like "4" that would always give the output of len(key_begin) as 1, can you explain if otherwise?